frontend/packages/data-portal/app/components/AnnotatedObjectsList.tsx

```diff
@@ -0,0 +1,75 @@
+import { Tooltip } from '@czi-sds/components'
+import Paper from '@mui/material/Paper'
+import { ReactNode } from 'react'
+
+import { ANNOTATED_OBJECTS_MAX } from 'app/constants/pagination'
+import { cns } from 'app/utils/cns'
+
+function List({
+  className,
+  children,
+}: {
+  className?: string
+  children: ReactNode
+}) {
+  return (
+    <ul className={cns('list-none flex flex-col gap-sds-xs', className)}>
+      {children}
+    </ul>
+  )
+}
```

Comment:
can we extract this under like `components/basic`? I could use something like this in my single dataset description PR